### PR TITLE
Updates to the Useful Windows Tips page

### PR DESCRIPTION
**Pull request by sthivaios**
Added 4 tips to the Useful Tips page.

Yes, not even a day has passed since #162 was merged but eh 🙃.

Anyway added the following : Voice Typing in Windows, Calculator always on top, Additional taskbar clocks, Clipboard history

Also added a little notes section at the bottom.

That's all :)

### CODE REVIEW

**Comment by sthivaios** [comment]
Keep in mind that that the vuepress config.js and the wiki index.md will conflict with #161.

**Review by FloatingMilkshake — COMMENTED** [review]
Some minor comments—otherwise, looks good to me

File: wiki/useful-windows-tips.md [hunk 1]
```diff
@@ -82,4 +82,68 @@ To do that, right-click on the columns in the Details view, and then choose More
...
+
+<img src="./img/useful-windows-tips/clipboard-history-list.png" alt="Screenshot of the clipboard history window showing text that was copied earlier." height=300px>
+
+You can click "Clear all" to empty the list. You can also click the pin icon next to an entry, to keep it pinned forever<sup>1</sup>.
+
+You can also click the three dots next to an entry, for more options for that particular entry.
+
+At the top of the window, you can also enter Emojis, GIFs, Analog Emojis, and other symbols.
+
+## Notes
```
[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:142]: @FloatingMilkshake i added the footnotes section cuz we might add more in the future idk

File: wiki/useful-windows-tips.md [hunk 2]
```diff
@@ -82,4 +82,68 @@ To do that, right-click on the columns in the Details view, and then choose More
...
+
+<img src="./img/useful-windows-tips/turn-on-clipboard-history.png" alt="Screenshot of the clipboard history window with the Turn On button visible." height=300px>
+
+After clicking on it, everything you copy will be logged there. You can then "go back in history" and paste something you copied earlier.
+
+<img src="./img/useful-windows-tips/clipboard-history-list.png" alt="Screenshot of the clipboard history window showing text that was copied earlier." height=300px>
+
+You can click "Clear all" to empty the list. You can also click the pin icon next to an entry, to keep it pinned forever<sup>1</sup>.
+
+You can also click the three dots next to an entry, for more options for that particular entry.
```
[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:138]: yeah i kinda repeated myself there lol

also, that comma on line 136 is WebStorm's fault

File: wiki/useful-windows-tips.md [hunk 3]
```diff
@@ -82,4 +82,68 @@ To do that, right-click on the columns in the Details view, and then choose More
 
 <img src="./img/useful-windows-tips/enable-more-columns-fileExplorer.png" alt="Screenshot of the window for adding more columns to the File Explorer details view." height=350px>
 
+### Voice typing
+
+::: tip Note
+To use Voice Typing, you need a microphone.
+:::
```
[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:89]: it's not (jk)

File: wiki/useful-windows-tips.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FloatingMilkshake) [reply wiki/useful-windows-tips.md:142]: Fair enough

[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:142]: heh

File: wiki/useful-windows-tips.md [hunk 4]
```diff
@@ -82,4 +82,68 @@ To do that, right-click on the columns in the Details view, and then choose More
...
+
+You can click "Clear all" to empty the list. You can also click the pin icon next to an entry, to keep it pinned forever<sup>1</sup>.
+
+You can also click the three dots next to an entry, for more options for that particular entry.
+
+At the top of the window, you can also enter Emojis, GIFs, Analog Emojis, and other symbols.
+
+## Notes
+
+<sup>1</sup>: Clipboard history automatically clears after restarting your computer. Also keep in mind that it only stores the last 25 objects.
```
[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:144]: > you could note whether pinned items count towards the 25?

uuuuhh, idk 💀

[the remark this answers is not in the document]
Comment (FloatingMilkshake) [reply wiki/useful-windows-tips.md:144]: As far as I can tell it doesn't seem to, it looks like the 25 is just for items that aren't pinned

[the remark this answers is not in the document]
Comment (sthivaios) [reply wiki/useful-windows-tips.md:144]: Alr thx.

**Comment by sthivaios** [comment]
@FloatingMilkshake 
> Aware of this, but it's an easy fix (and wasn't introduced by this PR :P)

yes smh but what if erisa or someone else was looking over this

File: wiki/useful-windows-tips.md [hunk 5]
```diff
@@ -82,4 +82,64 @@ To do that, right-click on the columns in the Details view, and then choose More
...
+
+You can click "Clear all" to empty the list. You can also click the pin icon next to an entry to keep it pinned forever<sup>1</sup>.
+
+You can also click the three dots next to an entry for more options.
+
+At the top of the window, you can also enter Emojis, GIFs, Analog Emojis, and other symbols.
+
+## Notes
+
+<sup>1</sup>: Clipboard history automatically clears after restarting your computer. Pinned items are not cleared. Pinned objects count towards the 25-object limit.
```
Comment (FloatingMilkshake) [inline wiki/useful-windows-tips.md:140]:
```suggestion
<sup>1</sup>: Clipboard history automatically clears after restarting your computer. Pinned items are not cleared, and do not count towards the 25-object limit.
```

Comment (sthivaios) [reply wiki/useful-windows-tips.md]: fixed

**Review by FloatingMilkshake — APPROVED** [review]

**Pull request merged**